cypress/support/commands.js

```diff
@@ -24,3 +26,43 @@ Cypress.Commands.add('awaitLoad', (divIds) => {
         .and('eq', '0');
   });
 });
+
+/**
+ * Cypress tests have two phases. In the first phase, each line is executed, top
+ * to bottom, as usual. However, any `cy.*` commands, like `cy.get()`, do not
+ * actually do anything when they execute. Instead, they enqueue a command into
+ * an internal Cypress "queue". After all lines have been executed, Cypress
+ * knows the full queue. At that point, the "Cypress phase" starts. Cypress
+ * actually executes each command in the queue. After each command completes, it
+ * will execute any `.then` blocks that were chained onto it. Thus, in the
+ * following program:
+ * ```
+ * console.log('message 1');
+ * cy.wrap(console.log('message 2')).then(() => console.log('message 4'));
+ * console.log('message 3');
+ * cy.get('#id').then(() => console.log('message 5'));
```

Comment:
can you add a cy.wrap(console.log(...)) between the third and fourth lines?

Reply:
Sure, and added even more.